### PR TITLE
Incorrect Information regarding VolumeSnapshot API corrected

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #34143

### CODE REVIEW

**Comment by NitishKumar06** [comment]
@sftim @reylejano @xing-yang This is the new PR I've raised . The earlier PR was having some wrong commits. So I raised this new PR with the changes suggested by @Sea-n

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
...
+* [VolumeSnapshot v1beta1 CRD will be removed](https://github.com/kubernetes/enhancements/issues/177). Volume snapshot and restore functionality for Kubernetes and the [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, moved to GA in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.20 and is now unsupported. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and [Volume Snapshot GA blog](/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/) for more information.
```
Comment (sftim) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:76]:
```suggestion
* [VolumeSnapshot v1beta1 CRD will be removed](https://github.com/kubernetes/enhancements/issues/177). Volume snapshot and restore functionality for Kubernetes and the [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, moved to GA in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.20 and will become unsupported with the v1.24 release. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and the [Volume Snapshot GA](/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/) blog article for more information.
```

Comment (NitishKumar06) [reply content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md]: @sftim The following changes have been made now.

**Comment by NitishKumar06** [comment]
Thank you @reylejano and @xing-yang :)

**Pull request merged**